procrastinate/sql/queries.sql

```diff
@@ -48,8 +48,17 @@ WHERE id IN (
 
 -- finish_job --
 -- Stop a job, free the lock and record the relevant events
+-- Old version, kept for backward compatibility reasons
```

Comment:
```suggestion
-- Deprecated, remove after 1.0.0
```

Reply:
No... It's actually not necessary to keep it.

queries.sql is at the version of the code, not the DB. We'll never need that.

Reply:
Oh yes. Good catch!